### ISSUE TITLE
Build docs for multiple tags, selectable via a dropdown

### ISSUE THREAD

**Issue by lolmaus** [cibernox/ember-cli-yuidoc#33]
It's a very common situation when an addon author has to maintain two or more addon versions in parallel.

Generating docs only for the latest version means abandoning all users of previous version. When a new version is just released, that means abandoning 100% of the users.

How do we generate documentation for multiple tags at once?

**Comment by cibernox** [comment]
Good point, I'm not really sure how to achieve this. I considered it at the beginning but given that I expected some churn as the component take its final shape.

Given that the docs are not generated automatically from the code, the only vague plan I have is to have some sort of badge/label in the page that is always updated with the version of the code this version of the code matches, and take advantage of the fact that I'm using ember-cli with redis, so all previous version of the docs are available. 

Just by exposing links to those previous snapshots, users should see previous versions of the guides.

I need to check if I can do that with https://www.pagefronthq.com/ or I should migrate to heroku (which I might do anyway since I'm working on adding fastboot)

**Comment by lolmaus** [comment]
You know how `ember-try` hot-swaps different versions of Ember and Ember Data while running an Ember CLI task?

Consider this:

1. Have a config file to indicate which branches/tags to generate docs for.
2. Have the `ember-cli-yui` CLI task check out different git branches/tags and run docs generation for each, storing in subfolders.
3. In the end, checkout the starting branch as if nothing happened.
4. Each doc build should contain a dropdown list referencing all branches and indicating the current one. (Not sure how to achieve this during the build, might require hacking into YUIDoc. Alternatively, we can manually replace a placeholder with a list on each build after all of them are complete.)

**Comment by cibernox** [comment]
Oh wait, I thought this issue was in a different project and now that I realized, my response is pure nosense.

**Comment by lolmaus** [comment]
So what do you say now that we understood each other?

**Comment by cibernox** [comment]
I say that I've been on a transoceanic flight for the last 12 hours and
nothing productive will come out of my brain today.

Tomorrow I'll read this issue again 😴
